Gaijinizer patch/Troops.txt

```diff
@@ -12240,9 +12236,9 @@ Take a good look!#DL
 観念して、快楽に溺れてしまうがよい♡
 > CONTEXT: Troops/298/pages/1/339/Dialogue/enemy17
 Hohoho... I've caught you, \\n[1].
-He can no longer leave his mistress's body...
-It is better for him to give up and indulge
-in the pleasure ♡#DL
+You can no longer escape our body...
+You should just give in and indulge in the
+pleasure. ♡#Ron
 > END STRING
```

Comment:
"body" to "bodies", unless they're specifically mentioning their body as one here.